spec/features/idv/account_creation_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature 'account creation after LOA3 request', idv_job: true do
+  include SamlAuthHelper
+  include IdvHelper
+
+  context 'successful IdV with same phone as 2FA' do
```

Comment:
One more quick question that shouldn't be a blocker. What is the purpose of this context block?